chrysler_pacifica_ehybrid_s_2019/docker-compose-background.yml

```diff
@@ -14,7 +14,7 @@
 
 services:
   web-ui:
-    image: ${DOCKER_ORG}/carma-web-ui:${DOCKER_TAG}
+    image: usdotfhwastol/carma-web-ui:latest
```

Comment:
Are there any updates to carma-web-ui that aren't in develop now, or that should be?

Reply:
there is no carma-web-ui updates needed anymore